spynnaker/pyNN/models/spike_source/spike_source_poisson_vertex.py

```diff
@@ -174,7 +174,7 @@ def get_resources_used_by_atoms(
 
         # build resources as i currently know
         container = ResourceContainer(
-            sdram=SDRAMResource(self.get_sdram_usage_for_atoms(vertex_slice)),
+            sdram=ConstantSDRAM(self.get_sdram_usage_for_atoms(vertex_slice)),
```

Comment:
Shouldn't this return a variable size as it uses recording?

Reply:
I admit I do not understand the poisson full. But if its need for sdram changes with longer runs than 100% agreed.
Exactly why I named the change from SDRAMResource to ConstantSDRAM to force the decision.

That said

Reply:
I have improved the poisson cost models significantly now